mlflow/pyfunc/scoring_server/__init__.py

```diff
@@ -559,35 +573,17 @@ def get_cmd(
     local_uri = path_to_local_file_uri(model_uri)
     timeout = timeout or MLFLOW_SCORING_SERVER_REQUEST_TIMEOUT.get()
 
-    # NB: Absolute windows paths do not work with mlflow apis, use file uri to ensure
-    # platform compatibility.
-    if not is_windows():
-        args = [f"--timeout={timeout}"]
-        if port and host:
-            address = shlex.quote(f"{host}:{port}")
-            args.append(f"-b {address}")
-        elif host:
-            args.append(f"-b {shlex.quote(host)}")
-
-        if nworkers:
-            args.append(f"-w {nworkers}")
-
-        command = (
-            f"gunicorn {' '.join(args)} ${{GUNICORN_CMD_ARGS}}"
-            " -- mlflow.pyfunc.scoring_server.wsgi:app"
-        )
-    else:
-        args = []
-        if host:
-            args.append(f"--host={shlex.quote(host)}")
+    args = [f"--timeout-graceful-shutdown {timeout}"]
```

Comment:
Let's be sure to list out common configs in the follow up documentation guide for the server update.